x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule/index.tsx

```diff
@@ -68,29 +68,21 @@ TagContainer.displayName = 'TagContainer';
 
 const StepAboutRuleComponent: FC<StepAboutRuleProps> = ({
   addPadding = false,
-  defaultValues,
+  defaultValues: initialState,
```

Comment:
One confusing behaviour of this step: I tested the overrides locally and realized that two of the possible overridable fields (`Severity override` and `Risk score override`) are "normal" settings -not hidden-, while two others (`Rule name override` and `Timestamp override`) are part of the "Advanced settings" submenu, which the user has to unfold to see.

When testing updating these overrides and refreshing the preview, I realized there is a confusing difference between these two groups of overrides. The first two are applied to the Rule Preview as soon as you update them and click on "Refresh" in the Rule Preview flyout. On the contrary, if you update the "advanced" overrides, and click on "Refresh" the previewed alerts won't be updated. You first have to click on the "Continue" button of the "About rule" step, and only then click on "Refresh". Only then will the Rule Preview be updated.

I think, considering all overrides live within the same step, that they should all share the same behaviour.

Reply:
Yeah, that is a good point. I need to add those few fields to be observed and thus the value to be used to affect Rule Preview immediately. Thanks for catching that!